test/test.ts

```diff
@@ -27,11 +26,11 @@ import mockSdp from './mock-sdp.json';
 // }
 
 class MockRtcPeerConnection extends EventTarget {
-  constructor () {
+  constructor() {
```

Comment:
if you didn't make this change manually, we need to adjust the linter. Our standard is to pad between the name and the params' parens.

Reply:
For future reference, we talked about this. Unless something has changed, it appears that we can't change this with Prettier: https://github.com/prettier/prettier/issues/11148